src/test/java/no/unit/nva/PublicationMapperTest.java

```diff
@@ -0,0 +1,81 @@
+package no.unit.nva;
+
+import static no.unit.nva.model.util.PublicationGenerator.getEntityDescription;
+import static no.unit.nva.model.util.PublicationGenerator.getFileSet;
+import static no.unit.nva.model.util.PublicationGenerator.getOrganization;
+import static no.unit.nva.model.util.PublicationGenerator.getProject;
+import static no.unit.nva.model.util.PublicationGenerator.getPublication;
+import static nva.commons.utils.JsonUtils.objectMapper;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import java.net.URI;
+import java.util.UUID;
+import no.unit.nva.api.CreatePublicationRequest;
+import no.unit.nva.api.PublicationResponse;
+import no.unit.nva.api.UpdatePublicationRequest;
+import no.unit.nva.model.Publication;
+import org.junit.jupiter.api.Test;
+
+public class PublicationMapperTest {
+
+    public static final URI SOME_URI = URI.create("http://example.org");
+    public static final String SOME_OWNER = "owner";
+    public static final ObjectNode SOME_CONTEXT = objectMapper.createObjectNode();
+
+    @Test
+    public void canMapCreatePublicationRequestToNewPublication() throws Exception {
```

Comment:
can map create publication request to new publication? 
can map create PublicationRequest to new Publication?
can MapCreate PublicationRequest to NewPublication?

try to keep the format methodReturnsOutputWhenInputIsSomething